packages/button/src/Button.story.tsx

```diff
@@ -68,19 +98,73 @@ export const Playground: StoryFn<ButtonProps> = ({
     {...args}
   />
 );
+LiveExample.parameters = {
+  chromatic: {
+    disableSnapshots: true,
+  },
+};
 
-export const BaseGreen = Playground.bind({});
-BaseGreen.args = {
-  variant: Variant.BaseGreen,
+export const Focused: StoryType<typeof Button> = LiveExample.bind({});
+Focused.play = (async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const button = canvas.getByRole('button');
+  await userEvent.click(button);
+}) as PlayFn<typeof Button>;
+
+export const LargeSize: StoryType<typeof Button> = () => <></>;
+LargeSize.parameters = {
+  generate: {
+    args: {
+      size: Size.Large,
+    },
+  },
 };
 
-export const LoadingState = Playground.bind({});
-LoadingState.args = {
-  isLoading: true,
+export const DefaultSize: StoryType<typeof Button> = () => <></>;
+DefaultSize.parameters = {
+  generate: {
+    args: {
+      size: Size.Default,
+    },
+  },
 };
 
-export const LoadingStateWithText = Playground.bind({});
-LoadingStateWithText.args = {
-  isLoading: true,
-  loadingText: 'Saving',
+export const SmallSize: StoryType<typeof Button> = () => <></>;
+SmallSize.parameters = {
+  generate: {
+    args: {
+      size: Size.Small,
+    },
+  },
+};
+
+export const XSmallSize: StoryType<typeof Button> = () => <></>;
+XSmallSize.parameters = {
+  generate: {
+    args: {
+      size: Size.XSmall,
+    },
+  },
+};
+
+export const Loading: StoryType<typeof Button> = () => <></>;
```

Comment:
Nothing shows up in this story in SB, is that correct?

Reply:
Yeah we disabled it to avoid issues with the spinner